src/e2e/resources/simple-temple-expected/booking/dao/dao.go

```diff
@@ -108,7 +108,7 @@ func (dao *DAO) ReadBooking(input ReadBookingInput) (*Booking, error) {
 
 // UpdateBooking updates the booking in the datastore for a given ID, returning the newly updated booking
 func (dao *DAO) UpdateBooking(input UpdateBookingInput) (*Booking, error) {
-	row := executeQueryWithRowResponse(dao.DB, "UPDATE booking SET WHERE id = $1 RETURNING id;", input.ID)
+	row := executeQueryWithRowResponse(dao.DB, "UPDATE booking SET WHERE id = $1 RETURNING id, createdBy;", input.ID)
```

Comment:
@xsanda is fixing so this never actually gets generated, since the UPDATE endpoint will be omitted higher up in the chain